### ISSUE TITLE
Adding class or inline style for children only 

### ISSUE THREAD

**Issue by anetk** [frontend-collective/react-sortable-tree#661]
# Reporting a Bug?

I want to style Child 1 to 4 differently from parent elements. Is there a way to do this?
How do I access children? 

* Parent
      * Child 1
      * Child 2

* Parent
      * Child 3
      * Child 4

**Comment by nreis11** [comment]
Yep, this is possible. In my example, I set the color based on the depth of the node, but you can further customize using classes. 

Example:
https://codesandbox.io/s/6x76m9037k?fontsize=14&hidenavigation=1&theme=dark
